resource-sharing-partners/src/test/java/test/utils/BasebibliotekGenerator.java

```diff
@@ -52,8 +53,13 @@ public class BasebibliotekGenerator {
     private final Collection<? extends Record> records;
 
     public BasebibliotekGenerator(List<RecordSpecification> specificationList) {
-        currentRid = BigInteger.ONE;
-        records = generateRecordsFromSpecificationList(specificationList);
+        this.currentRid = BigInteger.ONE;
+        this.records = generateRecordsFromSpecificationList(specificationList);
+    }
+
+    public BasebibliotekGenerator(Record... records) {
+        this.currentRid = BigInteger.ONE;
```

Comment:
this.currentRid blir jo ikke riktig i denne konstruktøren, men det har jo foreløpig ikke noe å si for generateRecordsFromSpecificationList blir aldri kalt. Kanskje sette den til null? Ikke viktig da

Reply:
Satte den til null